### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Felixnoo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,54 +63,53 @@ Now you can try to create a PVC. If it is created in a namespace that is not all
...
        1.key: String. Required. Currently supports selection through the "Name" and "Phase" fields.
        2.operator: String. Required. Currently supports selection through the "In" and "NotIn" fields.
        2.values: []String. Required. 
 
 
-# Example
+# Examples
 
-The next few examples of yaml may be helpful for you to design Accessor:
+The following few examples of yaml may be helpful for you to design your own accessor.
```
Comment (Felixnoo) [inline README.md:109]: If you want to capitalize `Accessor`, keep all `accessor` capitalized throughout the document. If not, do not capitalize here.

File: README.md [hunk 2]
```diff
@@ -192,10 +191,10 @@ spec:
             operator: "In"
             values: ["app1", "app2"]
 ```
-This requires nameSpace to have the key "app" label and the value in this array: ["val1", "val2"]
+This requires nameSpace to have the key "app" label and the value in this array: ["val1", "val2"].
```
Comment (Felixnoo) [inline README.md:194]: I wonder whether `val1` and `val2` here should be `app1` and `app2`?

File: README.md [hunk 3]
```diff
@@ -234,9 +233,9 @@ spec:
             operator: "In"
             values: ["owner1", "owner2"]
 ```
-You can create the pvc of csi-qingcloud in namespace which (have the key "app" label and in the value in ["app1"]) **and** (have the key "owner" label and the value in ["owner1", "owner2"])
+You can create the PVC of csi-qingcloud in the following namespace: (have the key "app" label and in the value in ["app1"]) **and** (have the key "owner" label and the value in ["owner1", "owner2"]).
 
-### Both FieldSelector and LabelSelector
+### Both fieldSelector and labelSelector
```
Comment (Felixnoo) [inline README.md:238]: Please double-check the changes I made here.

**Review by Felixnoo — COMMENTED** [review]
I updated this document. Please see my comments and double-check them.

File: README.md (hunk 2 above)
Comment (f10atin9) [reply README.md:194]: Right, I was careless. :sweat:

File: README.md (hunk 1 above)
Comment (f10atin9) [reply README.md:109]: Thanks for your advice . it is so helpful to me

**Comment by f10atin9** [comment]
Thanks for your patient review,  I appreciate it！

**Pull request merged**